Chore: jaeger guidance (#4923)
dapr/docs: daprdocs/content/en/operations/observability/tracing/otel-collector/open-telemetry-collector-jaeger.md

  cr.jaegertracing.io/jaegertracing/jaeger:2.11.0
```

You can also view the logs the jaeger container using: 

```bash
docker logs jaeger
```

> Next, create the following `config.yaml` file locally:

Reviewer: The other edit you can do here is say that in the development environment navigate to the local dapr folder (add this reference link https://docs.dapr.io/getting-started/install-dapr-selfhost/#step-5-verify-components-directory-has-been-initialized) and update the default `config.yaml` file with these changes.

Can I also suggest that in this article https://docs.dapr.io/getting-started/install-dapr-selfhost/ we add an "Optional Step 6. Configure JaegerV2 for local tracing instead of Zipkin" and then add a link to this article. That is to give people a choice of the local tracing experience.

Reviewer: Other things to update

"1. Create a configuration file (e.g., `tracing.yaml`)" - Should be ->"1. Create a configuration file (for example `tracing.yaml`)

2. Why install cert- manager? And if so, why is this setting `isSecure: false`. I would expect that we show how to install secure connect with a cert (hence cert-manager yes)?

3. Is this namespace below correct between the deployment and  when you say "opentelemetry-operator-system" namespace?

Confirm that all resources in the `opentelemetry-operator-system` namespace are ready.

**Deploy a Jaeger V2 instance with in-memory storage**:
   Apply the following configuration to create a Jaeger V2 instance:
   
   ```yaml
   apiVersion: opentelemetry.io/v1beta1
   kind: OpenTelemetryCollector
   metadata:
     name: jaeger-inmemory-instance
     namespace: observability

Author: >Why install cert- manager? And if so, why is this setting isSecure: false. I would expect that we show how to install secure connect with a cert (hence cert-manager yes)?

Cert-manager is required for the otel-operator webhook to communicate with the API server, but for the purpose of this guide, we can simplify the installation.and use the automatically generated self-signed certificate option by setting `admissionWebhooks.certManager.enabled` to `false` and `admissionWebhooks.autoGenerateCert.enabled `to `true`.  

https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator#tls-certificate-requirement

Author: >Is this namespace below correct between the deployment and when you say "opentelemetry-operator-system" namespace?

Yes, the Operator runs in `opentelemetry-operator-system`, and it manages the jeager instance deployment in `observability`.

#### Installation

> **Note:**In order for the API server to communicate with the webhook component of the operator, the webhook requires a TLS certificate that the API server is configured to trust. There are a few different ways you can use to generate/configure the required TLS certificate detailed in the [otel operator chart docs](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator#tls-certificate-requirement)

> For simplicity in this case we will be letting helm to create an automatically generated self-signed certificate,

Reviewer: ```suggestion
For simplicity you can use Helm to create an automatically generated self-signed certificate.
```

> ```bash

Reviewer: Did you mean to add something here in this bash statement, or is this redundant?

Author: redundant, thanks for the spot
